CMakeLists.txt

```diff
@@ -145,35 +145,15 @@ if(NOT SERVER_ONLY)
   target_include_directories(adaptyst PUBLIC ${Boost_INCLUDE_DIRS})
   target_include_directories(adaptystserv PUBLIC ${Boost_INCLUDE_DIRS})
 
-  find_library(LIBNUMA
-    NAMES numa
-    DOC "libnuma")
-
-  if(LIBNUMA)
-    message(STATUS "Found libnuma: ${LIBNUMA}")
-
-    find_path(LIBNUMA_INCLUDE
-      NAMES numa.h
-      PATH_SUFFIXES include
-      DOC "libnuma header directory"
-    )
-
-    if (LIBNUMA_INCLUDE)
-      message(STATUS "Found numa.h inside ${LIBNUMA_INCLUDE}")
-      set(LIBNUMA_AVAILABLE TRUE)
-    else()
-      message(STATUS "numa.h not found, compiling without libnuma support")
-      set(LIBNUMA_AVAILABLE FALSE)
-    endif()
-  else()
-    message(STATUS "libnuma not found, compiling without libnuma support")
-    set(LIBNUMA_AVAILABLE FALSE)
-  endif()
-
-  if(LIBNUMA_AVAILABLE)
+  FIND_PACKAGE(PkgConfig REQUIRED)
+  PKG_CHECK_MODULES(NUMA numa)
```

Comment:
To make the code style consistent, could you change ```FIND_PACKAGE``` and ```PKG_CHECK_MODULES``` to ```find_package``` and ```pkg_check_modules``` respectively?

Reply:
Done.